src/file_parsing.jl

```diff
@@ -80,22 +80,22 @@ function log_component!(data::Dict,names,component,dict_type)
     end
 end
 
-get_values(param_set::ParamDict{FT}, names) where {FT} = get_values(param_set.data, names, param_set.dict_type, get_parametric_type(param_set))
+get_values(param_set::ParamDict{FT}, names) where {FT} = get_values(param_set.data, names, param_set.dict_type)
 
-function get_values(data::Dict, names, dict_type, ret_values_type)
+function get_values(data::Dict, names, dict_type)
     
-    ret_values = ret_values_type[]
+    ret_values = []
```

Comment:
We need to enforce the `Float` type somewhere: before, we created e.g. a `Float64[]` or `Float32[]` array. Then upon reading it converts the read value to `Float64` or `Float32`. 

If you remove this here, you need to enforce it elsewhere.

Reply:
e.g. within the `push!(...)` operation